CONTRIBUTING.md

```diff
@@ -163,3 +163,154 @@ minor versions at this time.
 Examples:
 
 - Refactored how Checks are stored in Etcd, `sensu-backend migrate` is required to upgrade
+
+## Development
+
+Sensu is written in Go, and targets the latest stable release of the Go
+compiler. To work on Sensu, you will need the latest release of Go.
```

Comment:
This is not actually true. We work with the circleci/go:1.10 tag right now, which I believe tracks the latest in 1.10.